lib/src/container/skopeo.rs

```diff
@@ -55,6 +58,41 @@ pub(crate) fn spawn(mut cmd: Command) -> Result<tokio::process::Child> {
     cmd.spawn().context("Failed to exec skopeo")
 }
 
+pub(crate) async fn copy_to_oci(
+    src: &ImageReference,
+    destdir: impl AsRef<Utf8Path>,
+    tag: Option<&str>,
+) -> Result<String> {
+    impl_copy_to_oci(src, destdir.as_ref(), tag).await
+}
+
+pub(crate) async fn impl_copy_to_oci(
+    src: &ImageReference,
+    destdir: &Utf8Path,
+    tag: Option<&str>,
+) -> Result<String> {
+    let digestfile = tempfile::NamedTempFile::new()?;
+    let mut cmd = new_cmd();
+    cmd.stdout(std::process::Stdio::null()).arg("copy");
+    cmd.arg("--digestfile");
+    cmd.arg(digestfile.path());
+    cmd.arg(src.to_string());
+    let destarg = tag
+        .map(|t| format!("oci:{destdir}:{t}"))
+        .unwrap_or_else(|| format!("oci:{destdir}"));
```

Comment:
It looks like this was unprefixed before, although I think this new form is the correct one.
Not sure if it was manually checked for unexpected side-effects.

Reply:
OK, now that I look in fact the change here was actually just wrong.  I thought we were testing it in CI, but we're not.

So...let's do https://github.com/ostreedev/ostree-rs-ext/pull/297 first, then I'll fix this code...